pickhardtpayments/SyncSimulatedPaymentSession.py

```diff
@@ -154,22 +172,21 @@ def _disect_flow_to_paths(self, s, d):
                     G[src][dest][channel.short_channel_id]["flow"] += flow
             else:
                 # FIXME: cost is not reflecting exactly the piecewise linearization
-                # Probably not such a big issue as we just disect flow
+                # Probably not such a big issue as we just dissect flow
                 G.add_edge(src, dest, key=channel.short_channel_id, flow=flow,
                            channel=channel, weight=channel.combined_linearized_unit_cost())
         used_flow = 1
         channel_paths = []
 
         # allocate flow to shortest / cheapest paths from src to dest as long as this is possible
-        # decrease flow along those edges. This is a standard mechanism to disect a flow int paths
+        # decrease flow along those edges. This is a standard mechanism to dissect a flow into paths
         while used_flow > 0:
-            path = None
             try:
                 path = nx.shortest_path(G, s, d)
             except:
                 break
             channel_path, used_flow = self._make_channel_path(G, path)
-            channel_paths.append((channel_path, used_flow))
+            channel_paths.append(Attempt(channel_path, used_flow))
```

Comment:
since you renamed paths to Attempts we might want to also rename `chanenl_paths`?

Reply:
k